pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -471,10 +478,16 @@ func GatherConfigMaps(i *Gatherer) func() ([]record.Record, []error) {
 		records := make([]record.Record, 0, len(cms.Items))
 		for i := range cms.Items {
 			for dk, dv := range cms.Items[i].Data {
-				records = append(records, record.Record{Name: fmt.Sprintf("config/configmaps/%s/%s", cms.Items[i].Name, dk), Item: ConfigMapAnonymizer{v: []byte(dv), encodeBase64: false}})
+				records = append(records, record.Record{
```

Comment:
We are fetching configmaps from ns openshift-config, so I think it doesn't make sense to add the NS prefix